debian/files

```diff
@@ -0,0 +1 @@
+qubes-img-converter-vm_1.1.3-1_amd64.deb utils optional
```

Comment:
Absolutely not - it's generated during the build and should be removed by *clean* target. Can simply be deleted.